app/templates/start_PCA_service.xml

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.11"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.11 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.11/schedulerjob.xsd"
+    name="Start_PCA_Service" 
+    priority="normal"
+    onTaskError="continueJobExecution"
+     maxNumberOfExecution="2">
+  <genericInformation>
+    <info name="workflow.icon" value="/automation-dashboard/styles/patterns/img/wf-icons/Start_PCA_Service.png"/>
+  </genericInformation>
+  <taskFlow>
+    <task name="Start_PCA_Service" 
+    
+    onTaskError="cancelJob" >
+      <description>
+        <![CDATA[ This task allows to start a PCA service from the client side. It requires an existing SERVICE_ID. Otherwise an exception is raised. ]]>
+      </description>
+      <variables>
+        <variable name="SERVICE_ID" value="" inherited="true" />
+        <variable name="INSTANCE_NAME" value="" inherited="true" />
+      </variables>
+      <genericInformation>
+        <info name="task.icon" value="/automation-dashboard/styles/patterns/img/wf-icons/Start_PCA_Service.png"/>
+        <info name="task.documentation" value="https://doc.activeeon.com/latest/PCA/PCAUserGuide.html"/>
+      </genericInformation>
+      <inputFiles>
+        <files  includes="cloud-automation-service-client-8.2.0-SNAPSHOT.jar" accessMode="transferFromGlobalSpace"/>
+      </inputFiles>
+      <forkEnvironment >
+        <additionalClasspath>
+          <pathElement path="cloud-automation-service-client-8.2.0-SNAPSHOT.jar"/>
+        </additionalClasspath>
+      </forkEnvironment>
+      <scriptExecutable>
+        <script>
+          <code language="groovy">
+            <![CDATA[
+import org.ow2.proactive.pca.service.client.ApiClient
+import org.ow2.proactive.pca.service.client.api.ServiceInstanceRestApi
+import org.ow2.proactive.pca.service.client.model.ServiceInstanceData
+import org.ow2.proactive.pca.service.client.model.ServiceDescription
+
+println("BEGIN " + variables.get("PA_TASK_NAME"))
+
+/*********************************************************************************
+* THIS PART IS SERVICE SPECIFIC. IF YOU NEED TO MODIFY SOMETHING, DO IT HERE       *
+/********************************************************************************/
+// It is possible to start a PCA service using some variables. In that case, we retreive their values in this part.  
+//e.g. def MY_Service_VAR_1 = $variables_WORKFLOW_VAR_1
+/********************************************************************************/
+
```

Comment:
I think these headers could be standardized, but it is just about aesthetics. Some lines starting with "*" others with "//" or "// "

Reply:
you are right @luispinedamx. I will harmonize